android/expoview/src/main/AndroidManifest.xml

```diff
@@ -1,116 +1,120 @@
 
-    <application>
+  <application>
 
-        <!-- WHEN_VERSIONING_REMOVE_FROM_HERE -->
-        <!-- Scheduled notifications -->
-        <receiver
-            android:name="host.exp.exponent.notifications.ScheduledNotificationReceiver"
-            android:exported="false"/>
-
-        <receiver
-            android:name="host.exp.exponent.notifications.receivers.SchedulingTriggerReceiver"
-            android:exported="true">
-            <intent-filter>
-                <action android:name="android.intent.action.REBOOT"/>
-                <action android:name="android.intent.action.BOOT_COMPLETED"/>
-                <action android:name="android.intent.action.TIMEZONE_CHANGED"/>
-                <action android:name="android.intent.action.TIME_CHANGED"/>
-                <action android:name="android.intent.action.QUICKBOOT_POWERON"/>
-            </intent-filter>
-        </receiver>
+    <!-- WHEN_VERSIONING_REMOVE_FROM_HERE -->
+    <!-- Scheduled notifications -->
+    <receiver
+      android:name="host.exp.exponent.notifications.ScheduledNotificationReceiver"
+      android:exported="false" />
 
-        <!-- WHEN_VERSIONING_REMOVE_TO_HERE -->
+    <receiver
+      android:name="host.exp.exponent.notifications.receivers.SchedulingTriggerReceiver"
+      android:exported="true">
+      <intent-filter>
+        <action android:name="android.intent.action.REBOOT" />
+        <action android:name="android.intent.action.BOOT_COMPLETED" />
+        <action android:name="android.intent.action.TIMEZONE_CHANGED" />
+        <action android:name="android.intent.action.TIME_CHANGED" />
+        <action android:name="android.intent.action.QUICKBOOT_POWERON" />
+      </intent-filter>
+    </receiver>
 
-        <provider
-            android:name="androidx.core.content.FileProvider"
-            android:authorities="${applicationId}.provider"
-            android:exported="false"
-            android:grantUriPermissions="true">
-            <meta-data
-                android:name="android.support.FILE_PROVIDER_PATHS"
-                android:resource="@xml/provider_paths"/>
-        </provider>
+    <!-- WHEN_VERSIONING_REMOVE_TO_HERE -->
 
-        <provider
-            android:name="versioned.host.exp.exponent.modules.api.components.webview.RNCWebViewFileProvider"
-            android:authorities="${applicationId}.fileprovider"
-            android:exported="false"
-            android:grantUriPermissions="true">
-            <meta-data
-                android:name="android.support.FILE_PROVIDER_PATHS"
-                android:resource="@xml/provider_paths" />
-        </provider>
+    <provider
+      android:name="androidx.core.content.FileProvider"
+      android:authorities="${applicationId}.provider"
+      android:exported="false"
+      android:grantUriPermissions="true">
+      <meta-data
+        android:name="android.support.FILE_PROVIDER_PATHS"
+        android:resource="@xml/provider_paths" />
+    </provider>
```

Comment:
I'm not sure if this change should be in this PR ;)

Reply:
Wow, that's weird. Must be some side effect of rebasing/upmerging. Will take a look at that.

Reply:
Really, there is only change in formatting. But GH shows the diff quite weirdly.